lib/core/utils/get-ancestry.js

```diff
@@ -2,22 +2,24 @@ import getShadowSelector from './get-shadow-selector';
 
 function generateAncestry(node) {
   const nodeName = node.nodeName.toLowerCase();
-  const parent = node.parentElement;
-  if (!parent) {
```

Comment:
this check has now been removed.
so after this PR (axe-core 4.10.1) we started seeing 
```
TypeError: Cannot read properties of null (reading 'children')
    at generateAncestry (eval at evaluate (:234:30), <anonymous>:7617:70)
```

Reply:
Oh dang. I thought I had added a conditional operator to the children call, but looks like I removed it in a refactor. I'll get a fix in. Thanks for letting us know.

Reply:
https://github.com/dequelabs/axe-core/pull/4617

Reply:
It happens as a part of an accessibility check in a Playwright test.

It doesn't fail every time though. perhaps it happens during some transition / animation state.

After looking closer at the test in question, looks like we omitted `await` by mistake there - so the test continues running during the acessibility check. I assume it might be causing the issue to in some cases